File: Week-1/Problem-Set-1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    // prompt for input between 1 and 8
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    //
    for (int i = 0; i < height; i++)
    {
        for (int k = i + 1; k < height; k++)
        {
            printf(" ");
        }
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        printf("  ");

        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week-6/Problem-Set-6/sentimental-mario-more/mario.py
def main():
    Height = get_int()
    for i in range(Height):
        for k in range(i + 1, Height):
            print(" ", end="")
        for j in range(i + 1):
            print("#", end="")

        print("  ", end="")

        for j in range(i + 1):
            print("#", end="")
        print("", end="\n")


def get_int():
    while True:
        try:
            Height = int(input("Height: "))
            if 1 <= Height <= 8:
                return Height
        except ValueError:
            print("Not an integer")


main()
